profiles/killbill/src/main/java/org/killbill/billing/server/log/obfuscators/ObfuscatorConverter.java

```diff
@@ -44,10 +45,16 @@
  */
 public class ObfuscatorConverter extends ClassicConverter {
 
-    private final Collection<Obfuscator> obfuscators = ImmutableList.<Obfuscator>of(new ConfigMagicObfuscator(),
-                                                                                    new LoggingFilterObfuscator(),
-                                                                                    new PatternObfuscator(),
-                                                                                    new LuhnMaskingObfuscator());
+    private final Collection<Obfuscator> obfuscators = new ArrayList<Obfuscator>();
+
+    @Override
+    public void start() {
+        obfuscators.addAll(Arrays.asList(new ConfigMagicObfuscator(),
+                                         new LoggingFilterObfuscator(),
+                                         new PatternObfuscator(getOptionList()),
```

Comment:
Since we're at it, should we pass these options to all obfuscators? They all have some form of hardcoded keys that we might have to dynamically configure in the future (e.g. custom internal header in `LoggingFilterObfuscator`)?

Reply:
I guess the exception is the LuhnMaskingObfuscator. Since they are very similar to the PatternObfuscator, we can come back to add it easily when we need that extra flexibility.